frame/evm/src/lib.rs

```diff
@@ -506,6 +507,11 @@ pub mod pallet {
 	#[pallet::getter(fn account_codes)]
 	pub type AccountCodes<T: Config> = StorageMap<_, Blake2_128Concat, H160, Vec<u8>, ValueQuery>;
 
+	#[pallet::storage]
+	#[pallet::getter(fn account_codes_metadata)]
```

Comment:
better to remove the `pallet::getter` syntax, because it will be deprecated by upstream.